test/e2e/elasticsearch_token_propagation_test.go

```diff
@@ -119,6 +118,20 @@ func (suite *TokenPropagationTestSuite) TestTokenPropagationTest() {
 		Finish()
 	closer.Close()
 
+	client := &http.Client{
```

Comment:
Could this check be moved into a separate test - makes test failure reporting better.